### ISSUE TITLE
[roads] render foot/bicycle=official/designated on all highways !

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3968]
**[Submitted to the original trac issue database at 1.34pm, Thursday, 18th August 2011]**

Please render foot/bicycle=official/designated like foot/bicycle=designated (colours) on all highways keeping all other specifications.

Thanks

**Comment by tomhughes** [comment]
**Author: Ldp**
**[Added to the original trac issue at 1.15pm, Thursday, 15th September 2011]**

What exactly do you mean by this?

**Comment by tomhughes** [comment]
**Author: skyper**
**[Added to the original trac issue at 5.26am, Monday, 13th August 2012]**

Replying to [comment:1 Ldp]:
> What exactly do you mean by this?

Damn, I still do not get mails about ticket changes.

Basicly, I though about rendering red dots/strips for foot=official and blue for bicycle and a mixture for both. Does propably not work for all highway types but for pathes this is really needed !

**Comment by tomhughes** [comment]
**Author: AMB**
**[Added to the original trac issue at 5.39pm, Friday, 9th November 2012]**

I think that what the original bug reporter was requesting is the same type of rendering that I use for the custom tiles on my website.  An example of the change can be seen here: http://www.gedanken.org.uk/mapping/custom-maps/#H_1_3_4 on the page describing my custom tiles.  The Mapnik styles that I use can also be downloaded from the same page.

For example if the highway is foot=permissive or foot=designated then the footway style is drawn on top of the normal style (the same for designated bridleways or designated cycleways).  The overlayed style may need to be drawn narrower than usual to avoid totally hiding the underlying nature of the highway though (my rendering probably doesn't do that).

**Comment by tomhughes** [comment]
**Author: math1985**
**[Added to the original trac issue at 10.13pm, Saturday, 29th March 2014]**

I'm not sure what is exactly meant with such roads.

For example, we have 12 612 roads with the combination bicycle=designated and highway=residential. What does such tagging mean? With what traffic signs is such a combination indicated?

**Comment by tomhughes** [comment]
**Author: math1985**
**[Added to the original trac issue at 11.48am, Monday, 26th May 2014]**

The key highway=designated is much more popular, and it is not clear what exactly the difference with highway=official is. Before we can implement this, the semantics of both tags need to be worked out, agreed on, and documented.

I will therefore close this as wontfix.
